bin/install_uninstall.ml

```diff
@@ -142,7 +142,6 @@ module Special_file = struct
 end
 
 type copy_kind =
```

Comment:
there isn't really a difference between this type and `option` now, but it's up to you if you find the names more descriptive.

Reply:
I agree; but this also documents that we're actually using artifact subtitution in this case, and also #10302 will touch a bit that part so I expect a third variant to appear in there.